Project-Hotpot/Project-Hotpot/Algorithm/Genre.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Genre.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/26/22.
+//
+
+import Foundation
+//node object for Genre Queue implementation
+class Genre: Equatable {
```

Comment:
Hmm, there should be a Queue or a linked list implementation already part of the swift language, no? cc @toby1236

Reply:
You know, Swift may not actually have this built-in; the closest I can find is the separate Swift Collections package: https://github.com/apple/swift-collections